components/ledger/notary-worker-selection/src/main/kotlin/net/corda/ledger/notary/worker/selection/NotaryVirtualNodeSelectorService.kt

```diff
@@ -0,0 +1,15 @@
+package net.corda.ledger.notary.worker.selection
+
+import net.corda.v5.base.annotations.Suspendable
+import net.corda.v5.ledger.common.Party
+
+/**
+ * A service that is used to select the next notary virtual node for the given notary service.
```

Comment:
```suggestion
 * A service that is used to select a notary virtual node for the given notary service.
```

The current text still implies something about how the worker is selected.